script.js

```diff
@@ -4,18 +4,45 @@ const numColors = 8;
 const numTiming = 5;
 const numTops = 3;
 const earningRate = 3605;
-const numberFormatter = new Intl.NumberFormat('en', { style: 'currency', currency: 'USD', maximumFractionDigits: 0, minimumFractionDigits: 0 });
+const currencyFormatter = new Intl.NumberFormat('en', { style: 'currency', currency: 'USD', maximumFractionDigits: 0, minimumFractionDigits: 0 });
+const numberFormatter = new Intl.NumberFormat();
 let amount = 0;
 
 function createBubble(cost, text) {
   const left = Math.floor(Math.random() * 65);
   const aside = document.createElement("aside");
   aside.style.left = `${left + 5}vw`;
   aside.className = `color${Math.floor(Math.random() * numColors)} time${Math.floor(Math.random() * numTiming)} top${Math.floor(Math.random() * numTops)}`;
-  aside.innerHTML = `<b>${Math.floor(cost / earningRate)} seconds</b>:<br>${text}<br/><i>${numberFormatter.format(cost)}</i>`;
+  aside.innerHTML = `<b>${prettyPrintTime(Math.floor(cost / earningRate))}</b>:<br>${text}<br/><i>${currencyFormatter.format(cost)}</i>`;
   body.append(aside);
 }
 
+function prettyPrintTime(sec_num) {
+  const days = Math.floor(sec_num / 60 / 60 / 24); // minutes / hours / days
+  const hours = Math.floor((sec_num - (days * 24 * 60 * 60)) / 60 / 60); // subtract # days (in seconds) / minutes / hours
+  const minutes = Math.floor((sec_num - (days * 24 * 60 * 60) - (hours * 60 * 60)) / 60); // subtract # days and hours / minutes
+  const seconds = sec_num - (days * 60 * 60 * 24) - (hours * 60 * 60) - (minutes * 60) // subtract # days, hours, and minutes for remaining seconds
```

Comment:
I did notice that line 24 doesn't have a semicolon at the end whereas the preceding lines do. I'm not sure if there's any style guide @alvaromontoro adheres to, but if this PR ever gets picked up, for consistency's sake it might be worth sticking a `;` in.